### ISSUE TITLE
Website not reflecting releases after 1.3.5

### ISSUE THREAD

**Issue by cricket007** [dropwizard/dropwizard#2590]
The website seems not to be updated for newest releases

Ref https://github.com/dropwizard/dropwizard/commit/07cd33e22a89e985c61ad929cbda765fe1422241#commitcomment-31846823

**Comment by cricket007** [comment]
Should the "other versions" page also list the versions in between 1.3.4 and 1.3.8?

https://www.dropwizard.io/1.3.8/docs/about/docs-index.html

**Comment by arteam** [comment]
Right, https://github.com/dropwizard/dropwizard/blob/master/docs/source/about/docs-index.rst was not updated. I will change it in the 1.3.x branch and release as part of the 1.3.9 docs.

**Comment by cricket007** [comment]
Sounds good. 👍

And what about the 1.3.8 release notes? https://github.com/dropwizard/dropwizard/blob/master/docs/source/about/release-notes.rst

**Comment by arteam** [comment]
Good point. I've cherry-picked them onto master: https://github.com/dropwizard/dropwizard/commit/f000627a307d12280e39875da99caf685d1950d3

**Comment by cricket007** [comment]
Thanks! I'll open new issues for anything else I see.
